### ISSUE TITLE
Crash without AppDirectory Parameter

### ISSUE THREAD

**Issue by andrewzirkel** [birkett/srvany-ng#1]
Have been playing with your app, which is great and is exactly what I need.  It crashes if there isn't an AppDirectory parameter for the service.  Maybe update that in the readme or make a default.

Thanks!

**Comment by wqweto** [comment]
This works here

```
@echo off
sc create frpcsvc start= auto binPath= "%~dp0srvany-ng.exe"
reg add HKLM\SYSTEM\CurrentControlSet\Services\frpcsvc\Parameters /v Application /f /d "%~dp0frpc.exe"
reg add HKLM\SYSTEM\CurrentControlSet\Services\frpcsvc\Parameters /v AppDirectory /f /d "%~dp0."
reg add HKLM\SYSTEM\CurrentControlSet\Services\frpcsvc\Parameters /v AppParameters /f /d "-c %~dp0frpc.ini"
```

. . . but without `AppDirectory` registry value the service crashes indeed.

**Comment by birkett** [comment]
Confirmed and resolved in https://github.com/birkett/srvany-ng/commit/0cfffe4c4f10a241377c9eae3e5007a95a1f3e33

This fix will be included in an upcoming 1.1 release.
